pkg/config/config.go

```diff
@@ -59,6 +59,35 @@ type ThinpoolOptionsConfig struct {
 	XfsNoSpaceMaxRetries string `toml:"xfs_nospace_max_retries"`
 }
 
+type AufsOptionsConfig struct {
+	// MountOpt specifies extra mount options used when mounting
+	MountOpt string `toml:"mountopt"`
+}
+
+type BtrfsOptionsConfig struct {
+	// MinSpace is the minimal spaces allocated to the device
+	MinSpace string `toml:"min_space"`
+}
+
+type OverlayOptionsConfig struct {
+	// IgnoreChownErrors is a flag for whether chown errors should be
+	// ignored when building an image.
+	IgnoreChownErrors string `toml:"ignore_chown_errors"`
+	// MountOpt specifies extra mount options used when mounting
+	MountOpt string `toml:"mountopt"`
+	// Alternative program to use for the mount of the file system
+	MountProgram string `toml:"mount_program"`
+	// Do not create a bind mount on the storage home
+	SkipMountHome string `toml:"skip_mount_home"`
+}
+
+type ZfsOptionsConfig struct {
+	// MountOpt specifies extra mount options used when mounting
+	MountOpt string `toml:"mountopt"`
+	// Name of the ZFS File system
+	Name string `toml:"name"`
```

Comment:
should this be fsname or is it wrong in the manpage?

Reply:
fixed